packages/cinterop/build.gradle.kts

```diff
@@ -632,3 +637,13 @@ realmPublish {
             "'io.realm.kotlin:gradle-plugin:${Realm.version}' instead."
     }
 }
+
+tasks.named("clean") {
+    doLast {
+        delete(
+            // FIXME Is is too dangerous to delete something outside build/ ... should these libs
```

Comment:
Linux & Windows are built on CI https://github.com/realm/realm-kotlin/blob/master/Jenkinsfile#L695-L721 so only macOS is already being built under the `build` directory https://github.com/realm/realm-kotlin/blob/master/packages/cinterop/build.gradle.kts#L344-L347 only the artifacts are copied into the `jni/resources` for them to be packaged automatically by the `Jar` task.
What didn't work previously is deleting the build directory won't trigger the recompilation of the  JVM/C-API since the gradle outputs were tied to the `jni/resources` folder